fastapi/encoders.py

```diff
@@ -20,15 +20,41 @@ def jsonable_encoder(
         include = set(include)
     if exclude is not None and not isinstance(exclude, set):
         exclude = set(exclude)
+    return _jsonable_encoder(
```

Comment:
after this https://gist.github.com/podhmo/adebe386eaf298b5fb7a1689f57f027e#file-01benchmark-txt